digitalfabrik/entitlementcard
# 1202: Android invalid certificate

### Short description

This pr adds a valid `let's encrypt` certificate to support `https requests` for android versions `< 7.1.1.`
Additional information can be found in the comment (code)

### Proposed changes



- add and load certificate

### Testing
- use an android emulator with android 7 or below
- use production build of sozialpass or entitlement card
- check if you can reach the backend

### Resolved issues



Fixes: #1202
